.github/workflows/deploy-preview.yml

```diff
@@ -0,0 +1,44 @@
+name: Deploy PR Preview
+# Only run when a PR has the 'preview' label
```

Comment:
Can you please also add this to the README, i.e., an explanation of how to use this, and what link the resulting data will be served at.

Reply:
I added it in a separate pull request that's already merged.